ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-server-groups.j2

```diff
@@ -12,6 +11,7 @@ aaa group server {{ aaa_server_group.type }} {{ aaa_server_group.name }}
 {%                             set server_cli = server_cli ~ " vrf " ~ server.vrf %}
 {%                         endif %}
    {{ server_cli }}
+!
```

Comment:
In general we should but the `!` before a section, so any mix will produce a consistent output, and never two consequtive `!`. By moving all the AAA stuff into a sub template, we can handle it for the group.